### DISCUSSION TITLE
Why does the AppStore add a "app.java" argument when copying to clipboard?

### DISCUSSION THREAD

**Discussion by robin-a-meade** [jbangdev/jbang#1719] [Q&A] [answered]
Why does the [AppStore](https://www.jbang.dev/appstore/) add a `app.java` argument when copying to clipboard?

For example,
![image](https://github.com/jbangdev/jbang/assets/1343270/77e020b2-19e1-4fca-adae-37fff1c61e04)
When I click the clipboard icon, the following command is copied to my clipboard:
```
jbang hello@eugenp/tutorials~jbang app.java
```
What is the significance of the `app.java` argument?

**Comment by maxandersen** [comment, marked as answer]
no significance. that is a bug noone seem to have noticed before now :)

Pushed a fix - will take a bit before live to not be ratelimited.
